dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05350AddDotSaltClusterColumn.java

```diff
@@ -0,0 +1,99 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.util.Base64;
+import com.liferay.util.Encryptor;
+import com.liferay.util.EncryptorException;
+import io.vavr.control.Try;
+import java.sql.SQLException;
+
+public class Task05350AddDotSaltClusterColumn implements StartupTask {
+
+    private static final String SELECT = "SELECT cluster_id FROM dot_cluster";
+    private static final String UPDATE = "UPDATE dot_cluster SET cluster_salt = ? WHERE cluster_id = ? ";
+
+    private static final String POSTGRES_SQL_ADD_DOT_CLUSTER_SALT_COLUMN = " ALTER TABLE dot_cluster ADD cluster_salt VARCHAR(256)";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal " ALTER TABLE dot_cluster ADD cluster_salt VARCHAR(256)" appears 4 times in this file; the first occurrence is on line 22](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)